Added release notes for Media streaming agents (#24318)
newrelic/docs-website: src/content/docs/release-notes/streaming-browser-release-notes/streaming-browser-agent-dashjs-26-02-18.mdx

---
subject: Streaming Video & Ads for Browser
title: Media agent for Dash JS v4.0.1
releaseDate: '2026-02-18'
version: 4.0.1
metaDescription: Release notes for Media agent for Dash JS version 4.0.1
category: Dash JS
---

## Bug fixes

> ### contentBitrate Calculation Improvement

Reviewer: ```suggestion
### Improved contentBitrate calculation
```

> **Issue**: The `contentBitrate` attribute was reporting the target bitrate from the manifest rather than the actual measured throughput during playback.

Reviewer: ```suggestion
**Issue**: The `contentBitrate` attribute reported the target bitrate from the manifest instead of the actual measured throughput during playback.
```

> **Fix**: Updated the bitrate calculation method to use `getAverageThroughput()` from dash.js, which provides the measured average throughput representing actual content consumption rate during playback. This gives more accurate real-time bitrate measurements.

Reviewer: ```suggestion
**Fix**: Updated the bitrate calculation method to use `getAverageThroughput()` from dash.js. This captures the measured average throughput, providing a more accurate, real-time representation of the content consumption rate during playback.
```

### Implementation details

- **Primary**: Uses `player.getAverageThroughput('video')` to get measured throughput

> - **Fallback**:

Reviewer: ```suggestion
- **Primary logic**: Uses `player.getAverageThroughput('video')` to retrieve measured throughput
- **Fallback logic**:
```
